nodejs/examples/rabbitmq.js

```diff
@@ -0,0 +1,29 @@
+const config = require("platformsh-config").config();
+const amqp = require('amqplib/callback_api');
+
+const credentials = config.credentials('rabbitmq');
+
+const rabbit_connection = 'amqp://' + credentials.username + ':' + credentials.password + '@' + credentials.host + ':' + credentials.port;
+
+amqp.connect(rabbit_connection, function(error0, connection) {
```

Comment:
Does amqp have an async/await version?  We're trying to stick to that in all the examples, in part for consistency and in part because callback hell is, er, hell. :smile:

Reply:
I will take a look to see how this can be done.

I'm getting a `TypeError: Cannot read property 'createChannel' of undefined` when I made the switch to use the async function with await declarations.